__tests__/app.test.js

```diff
@@ -314,3 +314,22 @@ describe("GET /api/users", () => {
       });
   });
 });
+
+describe("GET /api/articles (sorting queries)", () => {
+  test("GET:200 sorts the articles by any valid column for ascending or descending", () => {
+    return request(app)
+      .get("/api/articles?sort_by=created_at&order=asc")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles).toBeSortedBy("created_at", { descending: false });
+      });
+  });
```

Comment:
It is a big jump to have this as the first test. 

We should have a test for a request with just a sort_by query on its own to ensure this works. Then another test for order on its own to ensure this works as well, and then having this test of combining these behaviours to ensure they work together should be at least the third test. 

These queries should work independently as well